compiler/test/suites/records.re

```diff
@@ -161,4 +161,13 @@ describe("records", ({test}) => {
       bar.foo = Some(foo)
     |},
   );
+  assertRun(
+    "export_import_record_issue_665",
+    {|
+      import { Foo } from "data"
+      export enum Bar { Baz(Foo<Number>) }
+      print(Baz({ bar: 1 }))
+    |},
+    "Baz(<record value>)\n",
```

Comment:
@ospencer any idea why the record value is opaque to the printer? I exported the record from the imported module.

Reply:
No, that's very odd. Very concerning. Shouldn't be related to this change though...

Reply:
Realized what's wrong here. Definitely unrelated.

Reply:
#1021